ITEventsTests/SearchParametersPresenterTests/SearchParametersPresenterTests.swift

```diff
@@ -0,0 +1,120 @@
+import XCTest
+@testable import ITEvents
+
+class SearchParametersPresenterTests: XCTestCase {
+    var searchParametersPresenter: ISearchParametersPresenter!
+    var view: (ISearchParametersView & ISearchParametersViewMock)!
+    var searchParametersService: ISearchParametersService!
+    
+    override func setUp() {
+        super.setUp()
+        
+        searchParametersService = SearchParametersService()
```

Comment:
Using the real implementation ```searchParametersService = SearchParametersService()``` we use in App is bad here.
The reason why is that if our "real implementation" will start have broken logic (working with an error) then we only want to have ```SearchParametersService``` related tests to be failed. We don't want these tests to be failed as well right? as here we are testing the presenter.

So if we want "UPOROTCA PO POLNOY" we should create and use one more mock for searchParametersService (```SearchParametersServiceMock```) here instead of using "real implementation":
```swift
var searchParametersServiceMock: SearchParametersServiceMock
```

And probably once start using that mock (```SearchParametersServiceMock```) we will be able to configure it a bit shorter:
```swift
searchParametersServiceMock.selectedTags = [Tag(id: 1, name: "JavaScript"),
                                            Tag(id: 2, name: "iOS")]
searchParametersServiceMock.otherTags = [Tag(id: 3, name: "Android"),
                                         Tag(id: 4, name: "Python")]
```

instead of how we configuring the "real implementation" now:
```swift
let selectedTags = [Tag(id: 1, name: "JavaScript"),
                    Tag(id: 2, name: "iOS")]
let otherTags = [Tag(id: 3, name: "Android"),
                 Tag(id: 4, name: "Python")]
searchParametersService.updateTags(selectedTags: selectedTags, otherTags: otherTags)
```